SynthDet V2 (#49)
Unity-Technologies/SynthDet: docs/NotebookInstructions.md

## Workflow

* Make sure [Docker Desktop](https://www.docker.com/products/docker-desktop) is installed.

* Open a command line and use the following command to download and run the Dataset Insights Docker image, and mount your local folders to it:

```

> docker run -p 8888:8888 -v <datasets_path>:/data -v <synthdet_notebook_path>:/tmp -t unitytechnologies/datasetinsights:latest

Reviewer: ```suggestion
docker run -p 8888:8888 -v <datasets_path>:/data -v <notebooks_path>:/tmp -t unitytechnologies/datasetinsights:latest
```

```

> In the above command, `<datasets_path>` is the path to the folder where all locally generated SynthDet datasets are stored, and `<synthdet_notebook_path>` is the location at which the `SynthDet_Statistics.ipynb` notebook file is located inside your SynthDet repository (`repository_root/Notebooks`). 

Reviewer: ```suggestion
In the above command, `<datasets_path>` is the path to the folder containing SynthDet datasets are stored, and `<notebooks_path>` is the full path to the `Notebooks` directory in this repository. 
```
I found the wording a bit confusing - was it supposed to point to the notebook itself or the directory containing it? Tried to clarify.

Author: It is supposed to point to the Notebooks folder. Thanks, reads better now :)

On OSX, `<datasets_path>` is:
```
/Users/username/Library/Application\ Support/UnityTechnologies/SynthDet/
```
And on Windows, it is:
```
C:\Users\username\AppData\LocalLow\UnityTechnologies\SynthDet
```
> :information_source: Remember to replace `username` in the above paths.

> :information_source: If using a Linux/Unix based OS, make sure the spaces in both paths are escaped with backslashes, as shown in the above example.

> > :information_source: If you get an error about the format of the command, try the command again **with quotation marks** around the folder mapping arguments, i.e. `"<datasets_path>:/data"`.

Reviewer: Might as well use quotation marks in the example, to remove 2/3 infos. It's standard practice and would allow spaces to not be escaped (I think?)

Author: can confirm it works with quotations and non escaped paths
